onebusaway-android/src/main/java/com/joulespersecond/oba/ObaAnalytics.java

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright (C) 2014
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.joulespersecond.oba;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.location.Location;
+import android.support.v4.app.Fragment;
+
+import com.google.android.gms.analytics.GoogleAnalytics;
+import com.google.android.gms.analytics.HitBuilders;
+import com.google.android.gms.analytics.Tracker;
+import com.joulespersecond.oba.elements.ObaRegion;
+import com.joulespersecond.seattlebusbot.Application;
+import com.joulespersecond.seattlebusbot.BuildConfig;
+import com.joulespersecond.seattlebusbot.R;
+
+import java.security.MessageDigest;
+
+
+/**
+ * Analytics class for tracking the app
+ *
+ * @author Cagri Cetin
+ */
+
+public class ObaAnalytics {
+
+    /**
+     * Users location accuracy should be less then 100m
+     */
+    private static final float LOCATION_ACCURACY_THRESHOLD = 100f;
+
+    /**
+     * To measure the distance when the bus stop tapped.
+     */
+    public enum ObaStopDistance {
+
+        DISTANCE_1("User Distance: 00000-00050m", 50),
+        DISTANCE_2("User Distance: 00050-00100m", 100),
+        DISTANCE_3("User Distance: 00100-00200m", 200),
+        DISTANCE_4("User Distance: 00200-00400m", 400),
+        DISTANCE_5("User Distance: 00400-00800m", 800),
+        DISTANCE_6("User Distance: 00800-01600m", 1600),
+        DISTANCE_7("User Distance: 01600-03200m", 3200),
+        DISTANCE_8("User Distance: 03200-∞m", 0);
```

Comment:
Let's change `∞` to `INFINITY` to avoid strange things happening with the infinity character.